react/src/tests/opReturn.test.ts

```diff
@@ -1,7 +1,144 @@
-import { stringToHex } from '../util/opReturn'
+import { exportedForTesting, parseOpReturnProps } from '../util/opReturn'
 
-describe('Sample', () => {
-  it ('', () => {
+const { stringToHex, prependNonceWithPushdata, generatePushdataPrefixedNonce, getDataPushdata } = exportedForTesting
+
+describe('stringToHex', () => {
+  it ('Simple conversion 1', () => {
```

Comment:
`it ('converts the PayButton lokad id from ascii to hex', () => {`

Reply:
Hmm that's not what it does tho.  I am assuming lokad is same as pushdata; but this function is used on the user data itself. It's also a generic function;  could be used to encode anything from string to hex using UTF8; not limited to ASCII

Reply:
> I am assuming lokad is same as pushdata

The lokad here is 50415900

> It's also a generic function; could be used to encode anything from string to hex using UTF8; not limited to ASCII

Right -- so you want unit tests named after various things that the function can do.

This particular unit  test is converting the PayButton lokad id from ascii to hex.

'Simple conversion 1' doesn't give any information to an external viewer of this codebase.